internal/backends/qt/qt_window.rs

```diff
@@ -2143,6 +2143,13 @@ impl WindowAdapterInternal for QtWindow {
         });
         ds.as_ref().get()
     }
+
+    fn focus_window(&self) {
+        let widget_ptr = self.widget_ptr();
+        cpp! {unsafe [widget_ptr as "QWidget*"] {
+            widget_ptr->activateWindow();
```

Comment:
You also need to call raise() to actually bringing it on top. Otherwise this just give the focus.

https://doc.qt.io/qt-6/qwidget.html#activateWindow
> If you want to ensure that the window is stacked on top as well you should also call [raise](https://doc.qt.io/qt-6/qwidget.html#raise)(). Note that the window must be visible, otherwise activateWindow() has no effect.

Reply:
Excellent, thanks for spotting.